charts/kube-master/templates/api.yaml

```diff
@@ -165,6 +182,16 @@ spec:
             - mountPath: /etc/kubernetes/bootstrap
               name: bootstrap
               readOnly: true
+          livenessProbe:
+            exec:
+              command:
+                - sh
+                - -c
+                - cat < /dev/null > /dev/tcp/{{ include "etcd.fullname" . }}/2379
+            initialDelaySeconds: 60
+            periodSeconds: 5
+            timeoutSeconds: 1
+            failureThreshold: 1
```

Comment:
@databus23 I also added a liveness probe to solve our cache problem if etcd is gone.

Reply:
Moved this to a separate PR.